addons/mail/static/src/owl/components/chatter_topbar/chatter_topbar.xml

```diff
@@ -4,29 +4,46 @@
     <t t-name="mail.component.ChatterTopbar" owl="1">
         <div class="o_ChatterTopbar">
             <div class="o_ChatterTopbar_leftSection">
-                <button class="btn btn-link o_ChatterTopbar_buttonSendMessage">
+                <button class="btn btn-link o_ChatterTopbar_buttonSendMessage"
+                        t-att-class="{
+                            'o-active': props.isComposerVisible and props.isComposerLog === false,
+                        }"
+                        t-att-disabled="props.isDisabled"
+                        t-on-click.prevent.stop="_onClickSendMessage">
                     Send message
                 </button>
-                <button class="btn btn-link o_ChatterTopbar_buttonLogNote">
+                <button class="btn btn-link o_ChatterTopbar_buttonLogNote"
+                        t-att-class="{
+                            'o-active': props.isComposerVisible and props.isComposerLog === true,
+                        }"
+                        t-att-disabled="props.isDisabled"
+                        t-on-click.prevent.stop="_onClickLogNote">
                     Log note
                 </button>
-                <button class="btn btn-link o_ChatterTopbar_buttonScheduleActivity">
+                <button class="btn btn-link o_ChatterTopbar_buttonScheduleActivity" t-att-disabled="props.isDisabled"
+                        t-on-click.prevent.stop="_onClickSchedule">
                     <i class="fa fa-clock-o"/>
                     Schedule activity
                 </button>
-                <button class="btn btn-link o_ChatterTopbar_buttonAttachments"
+                <button class="btn btn-link o_ChatterTopbar_buttonAttachments" t-att-disabled="props.isDisabled"
                         t-on-click.prevent.stop="_onClickAttachments">
                     <i class="fa fa-paperclip"/>
-                    <!-- TODO s'inspirer de threadCache pour afficher loader tant que tous les attachments sont pas fetchés -->
-                    <!-- TODO action loadThreadCache -->
-                    <span class="o_ChatterTopbar_buttonCount o_ChatterTopbar_buttonAttachmentsCount" t-esc="storeProps.attachmentsAmount"/>
+                    <t t-if="props.isDisabled or storeProps.areAttachmentsLoaded">
```

Comment:
Doesn't make sense that disabled topbar always show a counter.

Reply:
That's what master does ... Do we change that or do we stick with what is done ?

Reply:
I would change that.
IMO "disable" just means preventing user interactions, and the counter does not depend on that.
It makes sense to have spinner then show correct counter for a genuine thread, but temporary threads could have a dash ("-") instead of counter. What do you think?

Reply:
I think that the whole topbar makes no sense and should not be printed ... Is there any reason for us to print it as no action is available and no information is really accurate or useful ?

Reply:
I would say this is like that just for aesthetics. Initially was to avoid a blank right-column while creating a new record.
I guess the chatter is visually pleasant, so the purpose was to look like a chatter without being a full-fledge chatter.

Reply:
I see, knowing this, maybe we could just remove the attachment and followers counters ? I don't think it's great to have just an icon with "-" after it, don't you think ?

Reply:
Alex ? do you hear me ? Alex ? I think I lost you !